README.md

```diff
@@ -1,2 +1,422 @@
+
+Now you should see the demo app running on your android device.
+
+
+### 2. Setup ADBKeyboard
+In this example, we are going to automate the typing in the android device. In order to let askui fluently type as desired, we will use a virtual keyboard that handles the keyboard input via adb: [ADBKeyboard.apk](https://github.com/senzhk/ADBKeyBoard)
+
+1. Download the ADB-Keyboard package *(Important: Version 2.0)*: <br>[Link to Github Repo](https://github.com/senzhk/ADBKeyBoard/releases/tag/v2.0)
+2. Unzip it.
+3. Find your device:
+```bash
+# make sure that your android device is connected, and the USB debugging mode is enabled.
+adb devices
+```
+4. Install the ADBkeyboard on the device:
+```bash
+# inside ADBKeyBoard-2.0/
+adb -s <your device id> install ADBKeyboard.apk
+```
+5. Configure the ADB Keyboard:
+```bash
+# inside ADBKeyBoard-2.0/
+adb -s <your device id> shell settings put secure default_input_method com.android.adbkeyboard/.AdbIME
+```
+6. Enable the ADB Keyboard:
+```bash
+# inside ADBKeyBoard-2.0/
+adb -s <your device id> shell ime enable com.android.adbkeyboard/.AdbIME
+```
+7. To check if it is enabled: <br>Click on a textfield in an app and see if the `ADB Keyboard {ON}` notification is shown at the bottom of the screen.
+
+
+### 3. Setup askui
+1. Setup askui by following [these steps](https://docs.askui.com/docs/general/Getting%20Started/getting-started).
+2. We need to run the uiController directly with an extra argument to specify the runtime mode, as the current version of askui(ver. 0.3.2) doesn't provide the api for running it with the runtime argument yet. <br><br>From within your npm project path, go to the directory that contains the askui-ui-controller binary, and run `./askui-ui-controller -r android`
+```bash
+cd <YOUR_PROJECT_DIRECTORY>/node_modules/askui/dist/release/latest/<YOUR_PLATFORM>
+./askui-ui-controller -r android
+
+# for example, Mac OS
+cd node_modules/askui/dist/release/latest/darwin/askui-ui-controller.app/Contents/MacOS/
+./askui-ui-controller -r android
+
+# If you can't find the binary as described above,
+# then you might have askui freshly installed and haven't run it yet.
+# The binary gets downloaded as the askui test code runs.
+# Run the command below to run the askui test code:
+npx jest test/my-first-askui-test-suite.test.ts --config ./test/jest.config.ts
+```
+
+3. If you got them both(emulator and uiController) running, then we are ready to go for the UI automation.<br><br>If you are working with the test code from our [official docs](https://docs.askui.com/docs/general/Getting%20Started/writing-your-first-test/), then you need to deactivate few lines of the code in `test/helper/jest.setup.ts` that is running the uiController, because we are already running it manually in the previous step.
```

Comment:
both(emulator and uiController) -> both (emulator and UiController)

HTML is possible but not desirable in Markdown: Adding a blank line should give an extra paragraph.

few lines -> a few lines